Dockerfile

```diff
@@ -65,17 +65,12 @@ FROM slim AS anndata
 LABEL org.opencontainers.image.title "scpcatools-anndata"
 
 COPY docker/renv_zellkonverter.lock renv.lock
+ARG BASILISK_USE_SYSTEM_DIR=1
```

Comment:
This location should be fine, but just for safety, let's define this in the first build stage.

Reply:
> This location should be fine, but just for safety, let's define this in the first build stage.

Do you mean when building the slim image?